app/views/notifications/index.html.erb

```diff
@@ -1,19 +1,23 @@
 <div class="min-h-full flex flex-col justify-center py-12 sm:px-6 lg:px-8">
-  <% if @notifications.any? %>
-    <h1 class="mt-6 text-center text-3xl font-extrabold mb-5"><%= t(".title") %></h1>
-    <%= render PagyPaginatorComponent.new(id: "notifications", pagy: @pagy, url_array: [:notifications]) do |component| %>
-      <% component.loading_icon do %>
-        <%= render LoadingComponent.new %>
-      <% end %>
-      <%= render @notifications %>
+  <%= turbo_frame_tag "notifications" do %>
+    <% if @notifications.any? %>
+      <div class="my-6">
+        <h1 class="text-center text-3xl font-extrabold mb-5"><%= t(".title") %></h1>
+        <%= render PagyPaginatorComponent.new(id: "notifications", pagy: @pagy, url_array: [:notifications]) do |component| %>
```

Comment:
The Pagy nav at the bottom of this component adds a bunch of spacing to the bottom of this set of notifications. It makes it difficult to re-use this section of the index as a turboframe. What's the reason for it, and why is it hidden? Can we remove it? Can we re-arrange how pagy is setup so that we don't have this issue with decomposing the view?

Reply:
It's hidden because it should only show up when JavaScript is disabled. The Stimulus controller hides it on render. 

More than happy to decompose this if it makes sense. Same goes for the extra spacing - maybe that goes into the caller's code instead of the component.